src/main/java/mate/academy/model/Movie.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.util.Objects;
+
+@Entity
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String description;
+
+    public Movie() {
+    }

```

Comment:
you can delete it) when you don`t have other constructors, the constructor without parameters is created by default